Code/Editor/EditorFramework/Assets/Implementation/AssetCurator.cpp

```diff
@@ -1300,7 +1304,7 @@ void ezAssetCurator::ProcessAllCoreAssets()
             {
               if (name.GetHash() == 0 || pInfo->m_Info->m_sAssetTypeName == name)
               {
-                resReferences = ProcessAsset(pInfo, pAssetProfile, false);
+                resReferences = ProcessAsset(pInfo, pAssetProfile, ezTransformFlags::TriggeredManually);
```

Comment:
shouldn't this be TransformFlags::None ?

Reply:
Well my thinking was that if it is a core asset we can't live without it and should transform it.

Reply:
right, didn't see that